lightning/src/ln/functional_test_utils.rs

```diff
@@ -85,12 +85,13 @@ pub fn connect_block<'a, 'b, 'c, 'd>(node: &'a Node<'b, 'c, 'd>, block: &Block,
 	node.chain_monitor.chain_monitor.block_connected(&block.header, &txdata, height);
 	node.node.block_connected(&block.header, &txdata, height);
 	node.node.test_process_background_events();
+	node.blocks.lock().unwrap().push((block.header, height));
 }
 
 pub fn disconnect_block<'a, 'b, 'c, 'd>(node: &'a Node<'b, 'c, 'd>, header: &BlockHeader, height: u32) {
 	node.chain_monitor.chain_monitor.block_disconnected(header, height);
 	node.node.block_disconnected(header);
-	node.node.test_process_background_events();
```

Comment:
Was removing this intentional?

Reply:
Yes, though it happened int he wrong commit, I moved it to a later commit and since no previous tests relied on this behavior, removing this for `test_unconf_chan` to use `disconnect_block` is nice.

Reply:
Looks like the commit message for d1d1d1fb76fe4b3afb642392af33a09d3ba34dc3 doesn't reference a valid commit?

Reply:
Oh oops, its the immediately-preceeding "f" commit - it removes this hunk after "Add assertions for in-order block [dis]connection in ChannelManager"